sys/drv/rtl8139.c

```diff
@@ -32,18 +37,72 @@ static int rtl_reset(rtl8139_state_t *state) {
   return 1;
 }
 
-static int rtl8139_attach(device_t *dev) {
+static intr_filter_t rtl8139_intr(void *data) {
+  rtl8139_state_t *state = data;
+  // TODO: add enqueue routines
+  bus_write_2(state->regs, RL_ISR, RL_ISR_RX_OK);
+  return IF_FILTERED;
```

Comment:
It's a bad idea to always return `IF_FILTERED`. An interrupt event may consist of multiple handlers each of which is called during a call to `intr_event_run_handlers`. If the network card isn't the actual source of an interrupt, then you should return `IF_STRAY`.

Reply:
in the current PR I will return IF_STRAY - it will be improved in further PR.  - done